### DISCUSSION TITLE
檔案管理器用SMB協議傳輸只能跑到110MB

### DISCUSSION THREAD

**Discussion by amisjimmy** [linuxdeepin/developer-center#3518] [Q&A | 问答板块]
我用iperf3測過電腦和NAS之間都能跑到250MB以上，可是用檔案管理器傳檔穩定跑在110MB左右
想問下有可能是哪裡擋住，我完全沒頭緒

**Comment by BLumia** [comment]
作为对比，有尝试过使用 `gio` 或 `nautilus` 进行 samba 文件传输吗？如果有，效果也是如此吗？

**Reply by amisjimmy** [reply]
試過nautilus 也是差不多跑在90-100MB
gio 沒試過，也不會用

**Reply by BLumia** [reply]
猜测可能是 samba 协议本身或者 `gio` （注：深度档案管理器以及 `nautilus` 均基于 `gio` 进行网络文件系统的访问）的实现本身导致的不会跑满带宽。微软本身也对可能导致 SMB 传输缓慢有提供一些说明，可参见 https://learn.microsoft.com/en-us/windows-server/storage/file-server/troubleshoot/slow-file-transfer 。

ps. v20 所附带的 `gio` 模块版本相比而言较旧，或许 `v23` 上所附带的更高版本的 `gio` 也会改善相关问题，但这个大概要等 v23 发布后才比较方便实际进行测试。
